apps/web/pages/v2/success.tsx

```diff
@@ -352,14 +360,23 @@ export default function Success(props: SuccessProps) {
                               <p className="text-bookinglight">{bookingInfo.user.email}</p>
                             </div>
                           )}
-                          {bookingInfo?.attendees.map((attendee, index) => (
-                            <div
-                              key={attendee.name}
-                              className={index === bookingInfo.attendees.length - 1 ? "" : "mb-3"}>
-                              <p>{attendee.name}</p>
-                              <p className="text-bookinglight">{attendee.email}</p>
-                            </div>
-                          ))}
+                          {eventType.seatsHideAttendees
+                            ? bookingInfo?.attendees
+                                .filter((attendee) => attendee.email === email)
+                                .map((attendee) => (
+                                  <div key={attendee.name} className="mb-3">
+                                    <p>{attendee.name}</p>
+                                    <p className="text-bookinglight">{attendee.email}</p>
+                                  </div>
+                                ))
```

Comment:
Since emails are unique across attendees, we can filter to display just the current attendee if `seatsHideAttendees` is true